docs/eventlisteners.md

```diff
@@ -25,13 +25,13 @@ spec:
   triggers:
     - binding:
         name: pipeline-binding
-        params:
-          - name: message
-            value: Hello from the Triggers EventListener!
       template:
         name: pipeline-template
+      params:
+        - name: message
+          value: Hello from the Triggers EventListener!
 ```
 
 ## Parameters
-`EventListener`s can provide `params` which are passed to the `TriggerBinding`
-specified by the `binding.name`. Each parameter has a `name` and a `value`.
+`EventListener`s can provide `params` which are passed to the `TriggerTemplate`
```

Comment:
This is slightly confusing? The params are passed to the `TriggerTemplate` as specified, but it also sort of sounds like the params should be under the template. Maybe just me?

Reply:
The resource templates declare the parameters they need while the params here specify the values for those declared params. Usually, we'd pass those values in from the event via the binding or via default values.

Reply:
I rephrased it to hopefully make this more clear.